class.jetpack.php

```diff
@@ -561,6 +561,26 @@ static function delete_active_modules() {
 		self::update_active_modules( array() );
 	}
 
+	/**
+	 * Adds a hook to plugins_loaded at a priority that's currently the earliest
+	 * available.
+	 */
+	public function add_configure_hook() {
+		global $wp_filter;
+
+		$current_priority = has_filter( 'plugins_loaded', array( $this, 'configure' ) );
+		if ( false !== $current_priority ) {
+			remove_action( 'plugins_loaded', array( $this, 'configure' ), $current_priority );
+		}
+
+		$taken_priorities = array_map( 'intval', array_keys( $wp_filter['plugins_loaded']->callbacks ) );
+		sort( $taken_priorities );
+
+		$first_priority = array_shift( $taken_priorities );
+
+		add_action( 'plugins_loaded', array( $this, 'configure' ), $first_priority - 1 );
```

Comment:
We’ll probably want to make sure that we’re not doing -PHP_INT_MAX -1.

Reply:
Yes, fixed that in 3612a73, thanks!